lib/internal_metric/process.rb

```diff
@@ -22,6 +22,7 @@ class Process < Base
       major_gc_count: "Major GC operations by process",
       minor_gc_count: "Minor GC operations by process",
       total_allocated_objects: "Total number of allocateds objects by process",
+      scheduled_job_failures: "Number of recurring scheduled jobs that failed in a process"
```

Comment:
Per https://github.com/discourse/discourse/pull/17805, I don't think the metric only includes recurring scheduled job but can include normal job failures as well?

Reply:
true ... this is tricky ... I guess we name it "job_failures" will fix it up soon